examples/kiwix/manifests/persistence.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kiwix-data
+spec:
+  storageClassName: "###ZARF_STORAGE_CLASS###"
```

Comment:
Where does `STORAGE_CLASS` come from?

Reply:
This is a built in var - I copied it in from @jeff-mccoy 's original Kiwix example, but this should be a regular variable instead.

Reply:
NVM I forgot that this comes from `init` - basically init queries the k8s distro and selections a storage class or (optionally) it uses the override from the --storage-class flag.

https://github.com/defenseunicorns/zarf/blob/main/src/internal/cluster/state.go#L120

Reply:
Made that a bit clearer in a comment